### PR TITLE
Fix aws trial

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- set proper zone for region for AWS Trial plan

**Related issue(s)**
Fixes: https://github.com/kyma-project/control-plane/issues/924

### CODE REVIEW

File: components/kyma-environment-broker/cmd/broker/broker_suite_test.go [hunk 1]
```diff
@@ -367,7 +374,7 @@ func (s *BrokerSuiteTest) FinishProvisioningOperationByReconciler(operationID st
 func (s *BrokerSuiteTest) AssertProvisionerStartedProvisioning(operationID string) {
 	// wait until ProvisioningOperation reaches CreateRuntime step
 	var provisioningOp *internal.ProvisioningOperation
-	err := wait.Poll(pollingInterval, 2*time.Second, func() (bool, error) {
+	err := wait.Poll(pollingInterval, 2000*time.Second, func() (bool, error) {
```
Comment (wozniakjan) [inline components/kyma-environment-broker/cmd/broker/broker_suite_test.go:377]: nit: could be more readable as `33 * time.Minute`

File: components/kyma-environment-broker/cmd/broker/broker_suite_test.go [hunk 2]
```diff
@@ -381,7 +388,7 @@ func (s *BrokerSuiteTest) AssertProvisionerStartedProvisioning(operationID strin
 	assert.NoError(s.t, err)
 
 	var status gqlschema.OperationStatus
-	err = wait.Poll(pollingInterval, 2*time.Second, func() (bool, error) {
+	err = wait.Poll(pollingInterval, 2000*time.Second, func() (bool, error) {
```
Comment (wozniakjan) [inline components/kyma-environment-broker/cmd/broker/broker_suite_test.go:391]: nit: similar here, I'd personally write this as `33 * time.Minute` if we need such long polling

**Review by wozniakjan — COMMENTED** [review]
it looks like you may have `go fmt` configured differently from our pipelines
```
[0067] | ✗ some files are not properly formatted or contain not formatted imports. To repair run make imports
```

**Review by wozniakjan — APPROVED** [review]
/lgtm

File: components/kyma-environment-broker/cmd/broker/broker_suite_test.go (hunk 2 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/broker_suite_test.go]: fixed

File: components/kyma-environment-broker/cmd/broker/broker_suite_test.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/broker_suite_test.go]: fixed

**Pull request merged**